Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.txt

```diff
@@ -6,6 +6,16 @@
 //
 // Id: 2c55fe7a-b06f-4029-a5b9-c54a2320d7b8
 //
+// Severity: Medium
+//
+// QueryFrequency: 1h
+//
+// QueryPeriod: 30d
+//
+// AlertTriggerOperator: gt
+//
+// AlertTriggerThreshold: 0
```

Comment:
I think you want this to be 5, as that is your value in the script - let TotalEventsThreshold = 5;

Reply:
event threshold 5 used here to exclude events with low number of process executions count to reduce false positives however it is ambiguous in this case. Per discussion, we can wait to see how alerts are going to be configured using threshold to make changes here.